### PR TITLE
English: Release translation from 12.97.0 till 12.108.1

### PR DESCRIPTION
**Pull request by YokaiRick**
Hi,

i've translated all release notes from 12.97.0 till 12.108.1.
If i should change sth, just hit me up here.

Greetings

### CODE REVIEW

File: src/en/docs/releases.md [hunk 1]
```diff
@@ -5,8 +5,270 @@ Notes for indivudual misskey releases.
...
+- Misskey can no longer terminate HTTPS connections. @Johann150
+  - If you did not use a reverse proxy (e.g. nginx) before, you will probably need to adjust your configuration file and set up a reverse proxy. The https configuration key is no longer    recognized!
+
+### Improvements
+- Instance default theme can be set @syuilo
+- Allow mute to set expiration date @syuilo
+- Create a notification when a survey is closed @syuilo
+- Allow up to 16 additional profiles to be saved @syuilo
+- Add Pub&Sub to coalition chart @syuilo
+- Add Active to coalition chart @syuilo
```
Comment (Johann150) [inline src/en/docs/releases.md:32]:
```suggestion
- Allow up to 16 profile fields to be saved @syuilo
- Add Pub&Sub to federation chart @syuilo
- Add Active to federation chart @syuilo
```

File: src/en/docs/releases.md [hunk 2]
```diff
@@ -5,8 +5,270 @@ Notes for indivudual misskey releases.
...
+
+## 12.102.1 (2022/01/27)
+
+### Bugfixes
+- Fixed problem with chat not displaying
+
+## 12.102.0 (2022/01/27)
+
+### Note
+After the update, some custom pictograms may not be displayed. In this case, batch exporting the pictograms from the pictogram management page and then batch importing them again from the control panel will fix the problem.
```
Comment (Johann150) [inline src/en/docs/releases.md:160]: I think this would be clearer if you use "emoji" instead of "pictogram". Same for entries below in this release.

**Review by Johann150 — CHANGES_REQUESTED** [review]
Most of this looks great, thanks for translating! Still have some small corrections:

**Comment by YokaiRick** [comment]
also thanks for reviewing this :)

**Review by Johann150 — APPROVED** [review]

**Pull request merged**